concepts/recursion/about.md

```diff
@@ -1 +1,93 @@
-# About
+# Understanding Recursion in JavaScript
+
+Recursion is a powerful concept in programming that involves a function calling itself.
+It can be a bit tricky to grasp at first, but once you understand the fundamentals, it becomes a valuable tool in solving complex problems.
+In this tutorial, we'll explore recursion in JavaScript with easy-to-understand examples.
+
+## What is Recursion?
+
+Recursion occurs when a function calls itself, either directly or indirectly.
+It's similar to a loop, but it involves breaking a problem down into smaller, more manageable sub-problems.
```

Comment:
```suggestion
It's similar to a loop, but it may involve breaking a problem down into smaller, more manageable sub-problems.
```

This is often the case, but not necessarily.  Or we could say that it encourages breaking a problem down into smaller …